pkgs/development/libraries/science/math/nccl/default.nix

```diff
@@ -53,25 +50,18 @@ backendStdenv.mkDerivation rec {
 
   postFixup = ''
     moveToOutput lib/libnccl_static.a $dev
-
-    # Set RUNPATH so that libnvidia-ml in /run/opengl-driver(-32)/lib can be found.
-    # See the explanation in addOpenGLRunpath.
-    addOpenGLRunpath $out/lib/lib*.so
   '';
 
   env.NIX_CFLAGS_COMPILE = toString [ "-Wno-unused-function" ];
 
   enableParallelBuilding = true;
 
-  passthru = {
-    inherit cudaPackages;
-  };
-
   meta = with lib; {
     description = "Multi-GPU and multi-node collective communication primitives for NVIDIA GPUs";
     homepage = "https://developer.nvidia.com/nccl";
-    license = licenses.bsd3;
+    # Though the NCCL has a BSD3 license, it uses CUDA which is not free.
```

Comment:
This is an interesting one. In case of nccl we build from source and then link against the unfree cuda stuff. Nccl nix expression already won't evaluate, because it consumes cudaPackages. I think this is very different e.g. from torch-bin, where cuda bits are bundled in and there's nothing to prevent evaluation

Reply:
Yeah, I think the license should be kept as BSD3, because this is what NCCL is licensed by.

Reply:
Oh! Okay, that makes sense — will fix today.

Reply:
Fixed in newest commit

Reply:
Currently it still says `license = licenses.unfreeRedistributable;`

Reply:
Ugh. I forgot to do `git add`.

Reply:
Should definitely be correct now!